src/AuthContext.tsx

```diff
@@ -1,5 +1,11 @@
 import React, { createContext, useEffect, useRef, useState } from 'react' // eslint-disable-line
-import { fetchTokens, fetchWithRefreshToken, redirectToLogin, redirectToLogout, validateState } from './authentication'
+import {
+  fetchTokens,
+  fetchWithRefreshToken,
+  redirectToLogin,
+  redirectToLogout,
+  validateState,
+} from './authentication'
```

Comment:
I thought the CodeQL pipeline would pick up on this, but I checked out the PR locally, and prettier tells me this needs formatting.


```suggestion
import { fetchTokens, fetchWithRefreshToken, redirectToLogin, redirectToLogout, validateState } from './authentication'
```